samples/wifi/radio_test/src/nrf_wifi_radio_ficr_shell.c

```diff
@@ -0,0 +1,345 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* @file
+ * @brief NRF Wi-Fi radio test shell module
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr_util.h>
+#include <zephyr_fmac_main.h>
+#include <nrf_wifi_radio_test_shell.h>
+#include "ficr_prog.h"
+
+LOG_MODULE_REGISTER(otp_prog, CONFIG_WIFI_LOG_LEVEL);
+
+static void disp_location_status(char *region, unsigned int ret)
+{
+
+	switch (ret) {
+
+	case 0x00000000:
+		LOG_INF("%s Region is locked!!!\n", region);
+		break;
+	case 0x50FA50FA:
+		LOG_INF("%s Region is open for R/W\n", region);
+		break;
+	case 0xFFFFFFFF:
+		LOG_INF("%s Region is unprogrammed - program to enable R/W\n", region);
+		break;
+	default:
+		LOG_ERR("%s Region is in invalid state\n", region);
+		break;
+	}
+
+	LOG_INF("\n");
+}
+
+static void disp_fields_status(unsigned int flags)
+{
+	if (flags & (~QSPI_KEY_FLAG_MASK))
+		LOG_INF("QSPI Keys are not programmed in OTP\n");
```

Comment:
Please use surrounding `{ }` even for single-line blocks.

Reply:
This was there originally but I think checkpatch gave me warning so I changed it!! 
But sure, will change it back.

Reply:
For Zephyr (and thus NCS), you need to run checkpatch with `--ignore BRACES`, see https://docs.zephyrproject.org/3.1.0/contribute/guidelines.html#coding-style (first bullet).

Reply:
Done